src/functions/api-setup.js

```diff
@@ -0,0 +1,32 @@
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/UEH7RC3HNQw5hcmpfDEd/scores/';
+
+const scoreShow = (players) => {
+  const scoreSheet = document.querySelector('.high-list');
+  scoreSheet.innerHTML = '';
+  players.forEach((player) => {
+    scoreSheet.innerHTML += `<li class="high-score">${player.user} : ${player.score}</li>`;
+  });
+};
+
+async function getData() {
```

Comment:
- [ ] Please make sure to use arrow functions instead of the `function` keyword to create your functions.